packages/nextjs/src/common/pages-router-instrumentation/wrapGetServerSidePropsWithSentry.ts

```diff
@@ -36,7 +36,7 @@ export function wrapGetServerSidePropsWithSentry(
         sentryTrace,
       } = await (tracedGetServerSideProps.apply(thisArg, args) as ReturnType<typeof tracedGetServerSideProps>);
 
-      if (serverSideProps && 'props' in serverSideProps) {
+      if (typeof serverSideProps === 'object' && 'props' in serverSideProps) {
```

Comment:
Just to verify...do you want to include `null` (and other) values here? because `typeof null === 'object'`

You could check for `serverSideProps.constructor.name === "Object"` if you specifically look for an object.

Reply:
the joys of `any` :D If we want to make this safe I would do:

```js
serverSideProps && typeof serverSideProps === 'object' && 'props' in serverSideProps
```

If you make `serverSideProps: unknown` then this is what actually "works" in a type safe way, I believe.

Reply:
for some reason I fumbled this one and the errorgetinitialprops one